modules/EnsEMBL/Web/Component/Shared.pm

```diff
@@ -946,6 +946,7 @@ sub check_for_missing_species {
   my $db_key        = $args->{cdb} =~ /pan_ensembl/ ? 'DATABASE_COMPARA_PAN_ENSEMBL' : 'DATABASE_COMPARA';
   my $align_details = $species_defs->multi_hash->{$db_key}->{'ALIGNMENTS'}->{$align};
   my $species_info  = $hub->get_species_info;
+  my $url_lookup    = $species_defs->production_name_lookup;
```

Comment:
@ens-ap5 just making sure i understand the function production_name_lookup. It returns a hash with the species production name as key and value the species URL. is that correct?

Reply:
Yes